core/ui/src/views/SoftwareCenterAppInstances.vue

```diff
@@ -323,17 +323,23 @@
                       })
                     }}
                   </div>
-                  <template v-if="app.docs.relnotes_url">
-                    <span>
-                      <cv-link
-                        class="row icon-and-text"
-                        :href="app.docs.relnotes_url"
-                        target="_blank"
-                      >
-                        {{ $t("common.release_notes") }} <Launch20 />
-                      </cv-link>
-                    </span>
-                  </template>
+                  <div
+                    v-if="
+                      app.docs.relnotes_url &&
+                      isStableUpdateAvailable(app, instance)
+                    "
+                    class="row"
+                  >
+                    <cv-link
```

Comment:
We can use a standard `<a href="..."` here and remove `class="row icon-and-text"`